### PR TITLE
[Bayan] done

### PR DESCRIPTION
**Pull request by oOoOoOo0o**
accept this pr

### CODE REVIEW

**Review by carthy27 — APPROVED** [review]
WOHOOOOO

**Review by Blake1412 — CHANGES_REQUESTED** [review]
i disagree with these changes

**Review by Blake1412 — APPROVED** [review]
nvm

File: OBlock/contracts/StudentEnrolment.sol [hunk 1]
```diff
@@ -91,43 +91,40 @@ contract StudentEnrolment {
 
     /* Enroll a student */
     function enrollStudent(string memory moduleCode, string memory studentName) public {
-        // We can check if a module already exists by checking if its max capacity is greater than 0
```
Comment (Bourkupine) [inline OBlock/contracts/StudentEnrolment.sol:94]: remove these comments

**Review by Bourkupine — CHANGES_REQUESTED** [review]

File: OBlock/contracts/StudentEnrolment.sol (hunk 1 above)
Comment (oOoOoOo0o) [reply OBlock/contracts/StudentEnrolment.sol:94]: no

**Review by Bourkupine — APPROVED** [review]
Looks good, well done sir

**Comment by M-Byte480** [comment]
I think this needs another look. Reopening

File: OBlock/contracts/StudentEnrolment.sol (hunk 1 above)
Comment (M-Byte480) [reply OBlock/contracts/StudentEnrolment.sol:94]: > no
no

File: OBlock/src/enrolment.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ async function enrollStudentInModule() {
     const latestBlock = await web3.eth.getBlock('latest');
     const idx = latestBlock.number;
     const hash = latestBlock.hash;
-    const data = "{name: '${studentName}', module: '${moduleCode}'}"
+    const data = `{name: '${studentName}', module: '${moduleCode}'}`
```
Comment (M-Byte480) [inline OBlock/src/enrolment.js:11]: Good catch!

File: OBlock/contracts/StudentEnrolment.sol [hunk 3]
```diff
@@ -91,43 +91,40 @@ contract StudentEnrolment {
...
-//                string memory str = module.requisiteModules[i];
-//                if (comString(str, completed[j])) {
-//                    found = true;
-//                    break;
-//                }
-//            }
-//            require(found, "Student doesn't have requisite modules completed");
-//        }
-
-        require(comString(studentName, "Sam"));
```
Comment (samennis1) [inline OBlock/contracts/StudentEnrolment.sol:129]: Not allowed to use my name for GDPR reasons

Comment (oOoOoOo0o) [reply OBlock/contracts/StudentEnrolment.sol:129]: This pull request is closed bozo

File: OBlock/src/enrolment.js (hunk 2 above)
Comment (oOoOoOo0o) [reply OBlock/src/enrolment.js:11]: This pull request is closed dunderhead

**Comment by oOoOoOo0o** [comment]
> I think this needs another look. Reopening

This pull request is closed dingus

**Pull request merged**